tests/test_opds_import.py

```diff
@@ -549,12 +551,19 @@ def test_extract_link(self):
 
     def test_get_medium_from_links(self):
         audio_links = [
-            LinkData(href="url", rel="http://opds-spec.org/acquisition/", media_type="application/audiobook+json"),
+            LinkData(href="url", rel="http://opds-spec.org/acquisition/",
+                     media_type="application/audiobook+json;param=value"
+            ),
             LinkData(href="url", rel="http://opds-spec.org/image"),
         ]
         book_links = [
             LinkData(href="url", rel="http://opds-spec.org/image"),
-            LinkData(href="url", rel="http://opds-spec.org/acquisition/", media_type="application/epub+zip"),
+            LinkData(
+                href="url", rel="http://opds-spec.org/acquisition/",
+                media_type=random.choice(
+                    MediaTypes.BOOK_MEDIA_TYPES
+                ) + ";param=value"
+            ),
```

Comment:
This proves that any of the BOOK_MEDIA_TYPES is evidence that the work is BOOK_MEDIUM, even if there's a parameter attached to the media type.